artifactory/cli.go

```diff
@@ -2326,6 +2341,32 @@ func transferConfigCmd(c *cli.Context) error {
 	return nil
 }
 
+func dataTransferInstallCmd(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return cliutils.WrongNumberOfArgumentsHandler(c)
+	}
+	// Get the artifactory server
+	serverDetails, err := coreConfig.GetSpecificConfig(c.Args()[0], false, true)
+	if err != nil {
+		return err
+	}
+	installCmd := transferinstall.NewInstallDataTransferCommand(serverDetails)
+	// Optional flags
+	versionToInstall := c.String(cliutils.Version)
+	if versionToInstall != "" {
+		installCmd.SetInstallVersion(version.NewVersion(versionToInstall))
+	}
+	sourceFilesPath := c.String(cliutils.InstallPluginSrcDir)
```

Comment:
I think the command should not accept a version flag with a "path to files" flag. Only one of these flags should be accpetted by the command. See how we perform such input validations for other commands.